test/benchmarks/run.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+#------------------------------------------------------------------------------
+# Bash script to run optimizer performance tests.
+# ------------------------------------------------------------------------------
+# This file is part of solidity.
+#
+# solidity is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# solidity is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with solidity.  If not, see <http://www.gnu.org/licenses/>
+#
+# (c) 2022 solidity contributors.
+#------------------------------------------------------------------------------
+
+set -euo pipefail
+
+REPO_ROOT=$(cd "$(dirname "$0")/../../" && pwd)
+SOLIDITY_BUILD_DIR=${SOLIDITY_BUILD_DIR:-${REPO_ROOT}/build}
+
+solc="${SOLIDITY_BUILD_DIR}/solc/solc"
+benchmarks_dir="${REPO_ROOT}/test/benchmarks"
+chains_sol="${benchmarks_dir}/chains.sol"
+time_bin_path=$(which time)
+
+solc_command_legacy=("${solc}" --optimize --bin "${chains_sol}")
+solc_command_via_ir=("${solc}" --via-ir --optimize --bin "${chains_sol}")
+
+result_legacy_file=$(mktemp -t benchmark-legacy-XXXXXX.txt)
+result_via_ir_file=$(mktemp -t benchmark-via-ir-XXXXXX.txt)
```

Comment:
Please remember to delete them. It use `/tmp` all the time and it's really annoying when scripts just leave stuff there :) In other scripts we take care not to do that.

Reply:
I'll add a `reboot` at the end of the script, that should take care of it :smiling_imp:

Reply:
Turning it on and off always helps.